PWGUD/DataModel/UDTables.h

```diff
@@ -149,6 +151,18 @@ DECLARE_SOA_TABLE(UDCollisions, "AOD", "UDCOLLISION",
                   udcollision::NetCharge,
                   udcollision::RgtrwTOF);
 
+DECLARE_SOA_TABLE(SGCollisions, "AOD", "SGCOLLISION",
```

Comment:
UDCOllision and SGCollision are equal except for the column GapSide. I propose that you remove table SGCollision but add a table e.g. CillisionType which can be joined with UDCollisions and contains one column with the GapSide information.

Reply:
Done